src/assets/js/api-service.js

```diff
@@ -0,0 +1,34 @@
+// ignoring IIFE to test purposes - this should be really globals
+
+const AWS_IRMA_BACKEND_SERVER = 'https://f9emnttxd6.execute-api.ap-southeast-2.amazonaws.com/demo';
+
+const SECOND_PRIVATE_TOKEN = 'slU2raoR4f6q1eiVQbYuk5GGI3j2qaL25BdB6SSA';
```

Comment:
Might be a good idea to check the requester origin in the backend to make sure only our demo website can call the APIs. I'm gonna make that change.